.github/workflows/gradlew-test.yaml

```diff
@@ -23,11 +23,5 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
 
-      - name: Run Gradle Wrapper
-        run: chmod +x gradlew && gradle wrapper
-
       - name: Initialize Repo and Jars
         run: chmod +x initialize.sh && ./initialize.sh
-
```

Comment:
why don't we run tests anymore?

Reply:
We run ./gradlew build as part of the script which runs tests.

Reply:
And the reason to remove it is because we moved into two directories the GitHub workflow, which runs at the root of the project, would be confused. It's covered in the script regardless.

Reply:
then can you add a comment here and in `initialize.sh` saying just that + reference to docs saying `build` also runs `test`?

Reply:
Added to PR, added to GitHub workflow (comment + name change)